lbrynet/daemon/Daemon.py

```diff
@@ -1773,10 +1799,11 @@ def jsonrpc_stream_cost_estimate(self, uri, size=None):
         Get estimated cost for a lbry stream
 
         Usage:
-            stream_cost_estimate <uri> [<size> | --size=<size>]
+            stream_cost_estimate (<uri> | --uri=<uri>) [<size> | --size=<size>]
 
         Options:
-            <size>, --size=<size>  : stream size in bytes. if provided an sd blob won't be
+            --uri=<uri>      : (str) uri to use
+            --size=<size>  : (float) stream size in bytes. if provided an sd blob won't be
```

Comment:
please add (or remove) whitespace from these so the colons have matching indent levels (there are a few other instances)

Reply:
This will be automatically taken care by the scripts in the final output.